### ISSUE TITLE
1:m self relation

### ISSUE THREAD

**Issue by olaulau** [ikkez/f3-cortex#123]
hi ;

is it possible, and what would be the fieldConf of a 1:m self referring relationship ?
example : 
employee table has a manager field, which is a foreign key to employee->id
this describes a tree, which we should traverse in both direction (up and down).

thanks.

**Comment by ikkez** [comment]
Yes it's possible. Simply point it to the model itself. Manager is a belongs to one field config and another field, i.e. "manages" can be added to fieldConf that is a has-many configuration, pointing tongue manager-id field, which will contain all the child employees the manager will manage.
